instantiation/projectportal.md

```diff
@@ -0,0 +1,31 @@
+# Project Portal Service - User Guide
+
+## Introduction
+
+The Project Portal Service will provide a solution designed to support the needs of projects by offering a project
+website upon request. The portal, structured on a predefined base template, will offer the flexibility to further
+customize the overall user experience. Beyond the initial theming options, the service incorporates a Drupal Content
+Management System (CMS), empowering project members to perform advanced customizations and efficiently manage portal
+content.
+
+The instantiated project portal will have the option to work in synergy with other services offered within APEx. This
+collaborative approach enables projects to extend their portal by instantiating other APEx services such as a
+[STAC catalogue](catalog.qmd), a dashboard, a user forum, and more, and linking them to the main project portal,
+resulting in a dedicated, streamlined project ecosystem.
+
+## What does APEx offer?
```

Comment:
Can we mention the cost aspect? Project portals are sponsored via NoR or even apex budget, so significantly cheaper than other hosting solution? 
We also set up DNS, and keep things running beyond project lifetime?

Reply:
Good suggestion, I will add a part for the costs. I'm not sure on running it beyond the project lifetime as I don't think this is expected from APEx. Although results should remain available, I believe it is not APEx's goal to keep the project portals running.

Reply:
Added a section on the NoR and DNS - feel free to share your feedback. If OK, feel free to merge the branch and close the PR.